### ISSUE TITLE
Cannot use 'in' operator

### ISSUE THREAD

**Issue by deejroth** [Ragnarokkr/grunt-bumpx#6]
Javascript novice here, sorry if this is not actually a bug and ends up being my code.

When using the bump task in my Gruntfile, I get the the following error followed by the death of the script:

> Warning: Cannot use 'in' operator to search for 'src' in bower.json Use --force to continue.
> Aborted due to warnings.

I have tried calling the task using an alias and also explicitly calling it from the cli using `grunt bump:major`, `grunt bump:minor`, etc...Though I don't see where this would really play into the problem.

I am using grunt version 0.4.5, and grunt-bumpx version 0.2.0 on Debian Wheezy. The files specified in the Gruntfile do exists, and the version line exists in each file (they follow the Grunt and Bower standards). Other plugins I am using are: grunt-bowercopy, grunt-contrib-copy, and grunt-contrib-clean. I tried Googling for the problem, but the solutions do not seem to fit the issue. (Though, I am not a Javascript expert and could be incorrect!)

My Gruntfile config for grunt-bumpx:

``` javascript
bump: {
      'major': {
        options: {
          level: 'major',
          tabSize: 2
        },
        files: ['bower.json', 'package.json']
      },
      'minor': {
        options: {
          level: 'minor',
          tabSize: 2
        },
        files: ['bower.json', 'package.json']
      },
      'patch': {
        options: {
          level: 'patch',
          tabSize: 2
        },
        files: ['bower.json', 'package.json']
      }
    }
```

Hopefully this is not something I am doing as I have looked at your examples over and over and over again; to no avail.

**Comment by Ragnarokkr** [comment]
Hi @deejroth. Thanks for the feedback.

I'll check it out and let you know what I did figure out about this issue ASAP.

**Comment by Ragnarokkr** [comment]
The issue came due to an internal change about the way Grunt normalizes the paths between the various file declarations.

Now, to point grunt at only the source files (ignoring destinations) you have to use the `src` property instead of `files`. This is because using `files`, Grunt expects an array of `{src:..., dest:....}` objects.

I did also change documentation because of it.

**TLDR**

Use `src` instead of `files`.

**Comment by deejroth** [comment]
Thank you. I will change my config.
